### PR TITLE
minor refactor and Insertion Sort Algorithm

### PR DESCRIPTION
**Pull request by henrynnamani**

### CODE REVIEW

**Review by JerryJeager — COMMENTED** [review]
LGTM

File: Search.class
Comment (JerryJeager) [inline Search.class:1]: I don't think we need the binary file, can you remove this and add a .gitignore for all .class files? Thanks

**Review by JerryJeager — CHANGES_REQUESTED** [review]

**Comment by henrynnamani** [comment]
Aigt

File: Search.class
Comment (henrynnamani) [reply Search.class:1]: resolved

**Review by JerryJeager — APPROVED** [review]

**Pull request merged**